sopel/modules/ip.py

```diff
@@ -60,54 +59,37 @@ def _find_geoip_db(bot):
         else:
             LOGGER.warning(
                 'GeoIP path configured but DB not found in configured path'
             )
-    if (os.path.isfile(os.path.join(bot.config.core.homedir, 'GeoLiteCity.dat')) and
-            os.path.isfile(os.path.join(bot.config.core.homedir, 'GeoIPASNum.dat')) and
-            os.path.isfile(os.path.join(bot.config.core.homedir, 'GeoLiteCityv6.dat')) and
-            os.path.isfile(os.path.join(bot.config.core.homedir, 'GeoIPASNumv6.dat'))):
-        return bot.config.core.homedir
-    elif (os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoLiteCity.dat')) and
-            os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoIPASNum.dat')) and
-            os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoLiteCityv6.dat')) and
-            os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoIPASNumv6.dat'))):
+    if (os.path.isfile(os.path.join(config.core.homedir, 'GeoLite2-City.mmdb')) and
+            os.path.isfile(os.path.join(config.core.homedir, 'GeoLite2-ASN.mmdb'))):
+        return config.core.homedir
+    elif (os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoLite2-City.mmdb')) and
+            os.path.isfile(os.path.join('/usr/share/GeoIP', 'GeoLite2-ASN.mmdb'))):
         return '/usr/share/GeoIP'
     elif urlretrieve:
         LOGGER.warning('Downloading GeoIP database')
         bot.say('Downloading GeoIP database, please wait...')
-        geolite_city_url = 'http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz'
-        geolite_ASN_url = 'http://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum.dat.gz'
-        geolite_cityv6_url = 'http://geolite.maxmind.com/download/geoip/database/GeoLiteCityv6-beta/GeoLiteCityv6.dat.gz'
-        geolite_ASNv6_url = 'http://download.maxmind.com/download/geoip/database/asnum/GeoIPASNumv6.dat.gz'
-        geolite_city_filepath = os.path.join(bot.config.core.homedir, 'GeoLiteCity.dat.gz')
-        geolite_ASN_filepath = os.path.join(bot.config.core.homedir, 'GeoIPASNum.dat.gz')
-        geolite_cityv6_filepath = os.path.join(bot.config.core.homedir, 'GeoLiteCityv6.dat.gz')
-        geolite_ASNv6_filepath = os.path.join(bot.config.core.homedir, 'GeoIPASNumv6.dat.gz')
-        urlretrieve(geolite_city_url, geolite_city_filepath)
-        urlretrieve(geolite_ASN_url, geolite_ASN_filepath)
-        urlretrieve(geolite_cityv6_url, geolite_cityv6_filepath)
-        urlretrieve(geolite_ASNv6_url, geolite_ASNv6_filepath)
-        _decompress(geolite_city_filepath, geolite_city_filepath[:-3])
-        _decompress(geolite_ASN_filepath, geolite_ASN_filepath[:-3])
-        _decompress(geolite_cityv6_filepath, geolite_cityv6_filepath[:-3])
-        _decompress(geolite_ASNv6_filepath, geolite_ASNv6_filepath[:-3])
+        geolite_urls = [
+            'http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz',
+            'http://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz'
```

Comment:
Are these download URLs offered over HTTPS? Sopel should use secure connections to download external assets when possible.